tests/ExtruderPlanTest.cpp

```diff
@@ -308,39 +290,41 @@ TEST_P(ExtruderPlanPathsParameterizedTest, BackPressureCompensationHalf)
 {
     extruder_plan.paths = GetParam();
 
-    //Calculate what the flow rates were originally.
+    // Calculate what the flow rates were originally.
     std::vector<double> original_flows;
-    for(GCodePath& path : extruder_plan.paths)
+    for (GCodePath& path : extruder_plan.paths)
     {
         if (! shouldCountPath(path))
         {
-            continue; //Ignore travel moves.
+            continue; // Ignore travel moves.
         }
         original_flows.push_back(calculatePathWidth(path));
     }
-    const double original_average = std::accumulate(original_flows.begin(), original_flows.end(), 0.0) / original_flows.size();
+    const auto original_average = std::accumulate(original_flows.begin(), original_flows.end(), 0.0L) / original_flows.size();
 
-    //Apply the back pressure compensation with 50% factor!
+    // Apply the back pressure compensation with 50% factor!
     extruder_plan.applyBackPressureCompensation(0.5_r);
 
-    //Calculate the new flow rates.
+    // Calculate the new flow rates.
     std::vector<double> new_flows;
-    for(GCodePath& path : extruder_plan.paths)
+    for (GCodePath& path : extruder_plan.paths)
     {
         if (! shouldCountPath(path))
         {
-            continue; //Ignore travel moves.
+            continue; // Ignore travel moves.
         }
         new_flows.push_back(calculatePathWidth(path));
     }
-    const double new_average = std::accumulate(new_flows.begin(), new_flows.end(), 0.0) / new_flows.size();
-    //Note that the new average doesn't necessarily need to be the same average! It is most likely a higher average in real-world scenarios.
+    const auto new_average = std::accumulate(new_flows.begin(), new_flows.end(), 0.0L) / new_flows.size();
+    // Note that the new average doesn't necessarily need to be the same average! It is most likely a higher average in real-world
+    // scenarios.
```

Comment:
I'm fine with increasing the line length more, since we already use a longer length then most and I have a huge fancy widescreen monitor.
But we might also need to take into account people who read this on other devices and environments (web browsers, phones like I'm doing now)

I also think that the occasional "degradation" of something as subjectieve as readability doesn't out weights to benefits of lint tooling.
Because once we have come to an agreement over the values in clang-format, we will never have a code review again with comments like `indent with 4 space`

Reply:
There will always be places where it's easier to read than in other places. Line length is a trade-off. Making it too short makes all the code slightly harder to read on devices with big screens. Making it too long makes some lines of code much harder to read on devices with small screens.

The current code style has no limit because that works for most of our devices, since we tend to read code on small devices (like phones) in the browser, and Github applies line wrapping. We figured that if someone really does have only 120 columns space, they could enable line wrapping too.